terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/el/ElTest.java

```diff
@@ -398,7 +398,7 @@ public void test06_Query_Display() {
         // output 06_08 Test
         assertThat(
                 driver.findElement(
-                        By.xpath("//a[contains(@href, '?page=1&size=10&age=0&countries=&dateOfBirth=&main=false&name=TEST%255B%255D%2523%252B%253D%2526TEST')]"))
+                        By.xpath("//a[contains(@href, '?page=1&size=10&age=0&main=false&name=TEST%5B%5D%23%2B%3D%26TEST')]"))

```

Comment:
Probably ...
if property init value is `null`, current implementation is no problem.
e.g)
```java
class Foo {
    private String stringField;
    // ... (setter/getter)
}
```

But if it is any value, there is a possibility that the behaviors are difference.
e.g)
```java
class Foo {
    private String stringField = "any init value";
    // ... (setter/getter)
}
```

We should be check more patterns (e.g. when use the StringTrimmerEditor, etc..).